src/components/ContentSidebar/ActivityFeed/approval-comment-form/ApprovalCommentForm.js

```diff
@@ -227,7 +227,7 @@ class ApprovalCommentForm extends React.Component<Props, State> {
                         />
                         <aside
                             className={classNames('bcs-at-mention-tip', {
-                                'bcs-accessibility-hidden': isOpen
+                                'accessibility-hidden': isOpen
```

Comment:
May not have this class, probably copy it off from box-react-ui

Reply:
Appears that we are missing it. Will add `bcs-accessibility-hidden` to ApprovalCommentform.scss